### PR TITLE
chore(screenshotter): move coverage collection to Jest testing

### LINKED ISSUE #1885: Move screenshotter coverage collection to jest
After #1644, I've realized there is no need to collect coverage in screenshotter tests, increasing overheads on the screenshotter.

### PR DESCRIPTION
**Pull request by ylemkimon**
Fixes #1885. After #1644, I've realized there is no need to collect coverage in screenshotter tests, increasing overheads on the screenshotter.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -61,16 +61,6 @@ commands:
...
 
-  codecov:
-    parameters:
-      flag:
-        type: string
-    steps:
-      - run:
-          name: Upload code coverage reports to Codecov
-          # do not upload if screenshotter tests are skipped
-          command: ./node_modules/.bin/codecov -F "<< parameters.flag >>"
```
Comment (ylemkimon) [inline .circleci/config.yml]: I'm not sure there is need to separate Jest tests coverage and screenshotter coverage.

**Comment by kevinbarabash** [comment]
Why is this blocked on KaTeX/katex-fonts#62?

**Comment by ylemkimon** [comment]
@kevinbarabash Because screenshotter tests complain about missing metrics for spaces.

**Comment by ylemkimon** [comment]
I'm not sure why the coverage report is not showing.

**Comment by ylemkimon** [comment]
Closed in favor of #2324.

**Pull request closed without merging**